pkgs/by-name/su/surfer/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  fetchFromGitLab,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  wayland,
+  autoPatchelfHook,
+  libxkbcommon,
+  libGL,
+  libX11,
+  libXcursor,
+  libXi,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "surfer";
+  version = "0.2.0";
+
+  src = fetchFromGitLab {
+    owner = "surfer-project";
+    repo = "surfer";
+    rev = "v${version}";
+    hash = "sha256-C5jyWLs7fdEn2oW5BORZYazQwjXNxf8ketYFwlVkHpA";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    openssl
+    wayland
+    libxkbcommon
+    libGL
+    stdenv.cc.cc.libgcc
+  ];
+
+  # wayland and x11 libs required at runtime since winit uses dlopen
+  runtimeDependencies = [
+    openssl
```

Comment:
```suggestion
```
this is already dynamically linked. patching all rpaths with this is unnecessary